### ISSUE TITLE
Import/export buttons missing from options page

### ISSUE THREAD

**Issue by kalicki1** [privacy-tech-lab/gpc-optmeowt#287]
It seems that the import/export buttons are no longer in the options page after merging branch issue-175 into main. Not sure if it got broken along the way or otherwise. Concretely it looks like the buttons flash very quickly when I refresh the page, so I think they might be loaded and then immediately unloaded via some JS. 

Feel free to assign yourself if you pick this issue up; might be a good issue for @sophieeng as you're becoming acquainted with the frontend, though this isn't a high priority issue.

**Comment by SebastianZimmeck** [comment]
Yes, that is important to fix as we will be asking study participants to export their domain list and send it to us.
